kubernetes-services/work-scheduler/test/helpers/hooks.ts

```diff
@@ -0,0 +1,11 @@
+import { before } from 'mocha';
```

Comment:
another potential templates/example file - or better yet - just define this in one place and in the `.mocharc.yml` use it directly: `- ../../../test/helpers/test-hooks.ts`

Reply:
Not going to add a separate template project - will use this project as the template.